SI/Sem.02/Pract.02/Task7/Task7.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    char letter;
+    cout << "Enter a letter: ";
+    cin >> letter;
+    if (letter >= 'a' && letter <= 'z')
+        cout << "The big letter is " << (char)(letter - 32) << endl;
```

Comment:
We actually have not learnt casting yet :), just increment/decrement the letter before the cout